action.yml

```diff
@@ -29,7 +29,7 @@ runs:
   using: "composite"
   steps:
     - name: "Set up Python"
-      uses: "actions/setup-python@v5"
+      uses: "actions/setup-python@v5.1.0"
```

Comment:
What's the significance of pinning this version so precisely? Same question below.

Reply:
My assumption was that `v5` meant "always use latest v5 version` rather than `v5.0.0`. Is that not the case with GitHub actions?